Gerik/Crypters/Cezar/Caesar.java

```diff
@@ -70,30 +62,25 @@ public static void main(String[] args) {
                 System.out.println((char)27 +"[31m[!]Something went wrong!");
                 System.out.print((char)27 + "[37m");
             case "-h":
-                Info();
+                info();
                 System.exit(10);
         }
 
-        if(args.length != 3)
-        {
+        if(args.length != 3){
             System.out.print((char)27 +"[31m[!]Something went wrong!");
             System.out.println((char)27 + "[37m Use help(-h) for more info");
             System.exit(1488);
-        }
-        else
-        {
+        }else{
             char []text = args[1].toCharArray();
             step = Integer.parseInt (args[2]);
-            for(int i =0; i < text.length; i++)
-            {
-                letterIndex.setValue((int)text[i] - startIndex);
-                letterIndex.add(modifier * step);
-                text[i] = (char)(startIndex+letterIndex.getValue());
+            for(int i =0; i < text.length; i++){
```

Comment:
take a space between for params and gaps, like 
`for (int i =0; i < text.length; i++) {`